### ISSUE TITLE
🐛 Top nav active page gets marked wrong and is missing a11y styles

### ISSUE THREAD

**Issue by programmiri** [Aiven-Open/klaw-docs#141]
# Description

At the moment, there's no visible information for users which part of the top navigation they are on:

<img width="547" alt="bug-docs-2-whats-active" src="https://github.com/Aiven-Open/klaw-docs/assets/943800/8de5a7ec-a867-4f15-801c-6678431917b8">

We need to add accessible styles. 

Related to that, since "release" is  in the structure `/docs/releases`, both links are marked as active, so `aria-current` is set for both. Same would happen with styles. We need to make sure that works correctly.

**Comment by programmiri** [comment]
After some research -> it's not really possible to set the correct `active` nav item in our case. 

Our config looks like this:

```
          {
            type: "docSidebar",
            sidebarId: "tutorialSidebar",
            position: "left",
            label: "Docs",
          },
          { to: "/blog", label: "Blog", position: "left" },
          {
            to: "/docs/category/Releases",
            label: "Releases",
            position: "left",
          },
          {
            href: "https://github.com/aiven/klaw",
            label: "GitHub",
            position: "right",
          },
```

which means when the user is on url `/docs/category/Releases` it will show docs and releases as active. When the user is on url `//docs/category/Releases/somerelease` it will only show docs active. 

While there is a `activeBaseRegex` prop for items, it's not taken into account. There is an open PR on docusaurus (but over a year) and multiple issues, but I don't think that will be resolved anytime soon in an easy way to use for us. 

Since we want to restructure pages anyway, I would suggest that we add "Releases" not as part of "docs/" but as it's own "page" (same like blog). That way, this issue will resolve and it also reduces the amount of nav-items in docs. 

Tagging @harshini-rangaswamy  and @muralibasani  for that - not urgent.

**Comment by programmiri** [comment]
Resolved with https://github.com/Aiven-Open/klaw-docs/pull/179
